resources/todomvc/vanilla-examples/javascript-web-components/dist/components/todo-list/todo-list.component.js

```diff
@@ -33,8 +33,12 @@ class TodoList extends HTMLElement {
     }
 
     addItem(entry) {
+        const { id, title, completed } = entry;
         const element = new TodoItem();
-        Object.keys(entry).forEach((key) => element.setAttribute(key, entry[key]));
+
+        element.setAttribute("itemid", id);
+        element.setAttribute("itemtitle", title);
+        element.setAttribute("itemcompleted", completed);
```

Comment:
Can't you set the properties directly here?
The `connectedCallback` should take care of updating them, no?

Reply:
I do want also to have the id, title, completed attributes on the custom element. I use these attributes to check on status of the items. 

with the attributes: 
```
<todo-item dir="ltr" itemid="jqT4S4j6I6qSuy3SUUgBo" itemtitle="hi" itemcompleted="false" data-priority="4" style="display: block;"></todo-item>
```

without: 
```
<todo-item dir="ltr" data-priority="4" style="display: block;"></todo-item>
```

Reply:
ah right, make sense.